test/plugin/test_out_google_cloud.rb

```diff
@@ -1297,6 +1345,51 @@ def setup_logging_stubs
     end
   end
 
+  class GRPCLoggingMockService < Google::Logging::V1::LoggingService::Service
+    attr_reader :requests_received
+
+    def initialize
+      super
+      @requests_received = []
+    end
+
+    def write_log_entries(request, _call)
+      @requests_received << request
+      WriteLogEntriesResponse.new
+    end
+
+    def list_logs(_request, _call)
```

Comment:
Looks like the root cause of my confusion was a bug in gRPC Ruby (https://github.com/grpc/grpc/issues/9033). Let's keep the code as it stands now, but add a TODO with a reference to that issue to remove the dummy methods when that's fixed.